cluster-operator/src/main/java/io/strimzi/operator/cluster/model/VolumeUtils.java

```diff
@@ -405,4 +407,100 @@ private static String getVolumeNameHashStub(String name)   {
             throw new RuntimeException("Failed to get volume name SHA-1 hash", e);
         }
     }
+
+    /**
+     * Creates the Client Secret Volume
+     * @param tls    Kafka Client tls object
+     * @param volumeList    List where the volumes will be added
+     * @param isOpenShift   Indicates whether we run on OpenShift or not
+     */
+    protected static void createSecretVolume(ClientTls tls, List<Volume> volumeList, boolean isOpenShift) {
+        createSecretVolume(tls, volumeList, isOpenShift, null);
+    }
+
+    /**
+     * Creates the Client Secret Volume
+     * @param tls    Kafka Client tls object
+     * @param volumeList    List where the volumes will be added
+     * @param isOpenShift   Indicates whether we run on OpenShift or not
+     * @param alias   Alias to reference the Kafka Cluster
+     */
+    protected static void createSecretVolume(ClientTls tls, List<Volume> volumeList, boolean isOpenShift, String alias) {
+        if (tls != null) {
+            List<CertSecretSource> trustedCertificates = tls.getTrustedCertificates();
+
+            if (trustedCertificates != null && trustedCertificates.size() > 0) {
+                for (CertSecretSource certSecretSource : trustedCertificates) {
+                    addSecretVolume(volumeList, certSecretSource, isOpenShift, alias);
+                }
+            }
+        }
+    }
+
+    /**
+     * Creates the Volumes used for authentication of Kafka client based components, checking that the named volume has not already been
+     * created.
+     *
+     * @param volumeList    List where the volume will be added
+     * @param certSecretSource   Represents a certificate inside a Secret
+     * @param isOpenShift   Indicates whether we run on OpenShift or not
+     * @param alias   Alias to reference the Kafka Cluster
+     */
+    private static void addSecretVolume(List<Volume> volumeList, CertSecretSource certSecretSource, boolean isOpenShift, String alias) {
+        String volumeName = alias != null ? alias + '-' + certSecretSource.getSecretName() : certSecretSource.getSecretName();
+        // skipping if a volume with same name was already added
+        if (!volumeList.stream().anyMatch(v -> v.getName().equals(volumeName))) {
+            volumeList.add(VolumeUtils.createSecretVolume(volumeName, certSecretSource.getSecretName(), isOpenShift));
+        }
+    }
+
+    /**
+     * Creates the Client tls encrypted Volume Mounts
+     *
+     * @param tls    Kafka Client tls object
+     * @param volumeMountList    List where the volume mounts will be added
+     * @param tlsVolumeMountPath   Path where the TLS certs should be mounted
+     */
+    protected static void createSecretVolumeMount(ClientTls tls, List<VolumeMount> volumeMountList, String tlsVolumeMountPath) {
```

Comment:
It feels like the signature should be consistent with `addSecretVolume`, so:

1. Should the `volumeMountList` be the first parameter?
2. Should it take `CertSecretSource`, rather than `ClientTls`?